contracts/truster/FixedTrusterLenderPool.spec

```diff
@@ -0,0 +1,22 @@
+
+using DamnValuableToken as token;
+
+methods{
+    function token.allowance(address, address) external returns (uint256) envfree;
+    function _.execute(bytes) external => DISPATCHER(true);
+}
+
+/*
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Pool Allowance Always Zero                                                                                          │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+
+invariant poolAllowanceAlwaysZero(address user)
+    token.allowance(currentContract, user) == 0
+    {
+        preserved with (env e)
+        {
+            require e.msg.sender != currentContract;
+        } 
+    }
```

Comment:
it should be the same spec for both version,
can we just have `execute(bytes)` as part of the dispatching options

Reply:
Done